src/components/wallet/Newbie.svelte

```diff
@@ -6,6 +6,8 @@
   
   init_preferences();
 
+  import { getLocaleFromNavigator, setupI18n, _ } from "../../lang/i18n";
+  setupI18n({ withLocale: getLocaleFromNavigator() });
```

Comment:
this two added lines must be removed because the previous function is taking care of it.